bin/sync-all.sh

```diff
@@ -6,6 +6,12 @@ if [ ! -e ./manage.py ]; then
     cd $script_parent_dir
 fi
 
+# for heroku demos
```

Comment:
Note: this is here because Heroku doesn't currently support fetching submodules for automated deployments from github. I think this is fine for demos, and we want to move away from submodules anyway.